index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./index.css" />
+    <title>Awesome books-ES6</title>
+  </head>
+  <body>
+    <div class="main-container">
+      <header>
+        <nav class="container">
+          <p class="logo">AwesomeBook</p>
+          <ul>
+            <li class="listBtn">List</li>
+            <li class="add-new-btn">Add new</li>
+            <li class="contact">Contact</li>
+          </ul>
+        </nav>
```

Comment:
Good job working on the ES6 😃, but you need to fix some parts 👍.

- [ ]  Currently your `nav links` doesn't work and I can't see the list of books, kindly, fix it 👍 
- [ ] Also the current date doesn't display in the navigation bar, kindly fix it 👍 

![books](https://user-images.githubusercontent.com/57408419/227995449-79435e51-fef7-4512-8a52-5b2a1412628f.JPG)

Reply:
Have you installed the packages correctly? @joel-kalema

Reply:
yes i did @KUBAHO3